odoo/models.py

```diff
@@ -3125,6 +3125,9 @@ def get_field_translations(self, field_name, langs=None):
 
         return translations, context
 
+    def _get_lang_base(self):
```

Comment:
docstring

Reply:
nitpicking, I would had called the method `_get_base_lang` given the `base_url` method ("base" before the nun), a quick grep on my machine shows that usually we write `base_x` instead of `x_base`